### ISSUE TITLE
Handle Connection Loss

### ISSUE THREAD

**Issue by Bera0422** [cmpe-DroneLab/GeoFly3D#20]

**Comment by Bera0422** [comment]
Most of the cases are fixed. However, on May 7 Test we encountered a situation where the moveTo command did not execute at all. Interestingly that occurred only once. That part should be reviewed.

**Comment by Bera0422** [comment]
I think we can close this issue. If a significant problem arises, we can open a new issue.
